src/arch/ofi/machine-onesided.C

```diff
@@ -232,18 +243,29 @@ void LrtsIssueRget(NcpyOperationInfo *ncpyOpInfo) {
     req->size     = ncpyOpInfo->ncpyOpInfoSize;
     req->callback = send_short_callback;
     req->data.short_msg = ncpyOpInfo;
-
-    ofi_send(ncpyOpInfo,
+    // in CXI we cannot just send this unregistered thing
+    //
+#if CMK_OFI_CXI
```

Comment:
Where is `CMK_OFI_CXI` defined?

Reply:
That was replaced by CMK_CXI, due to it being impractical to get CMK_OFI_CXI to play nice with the cmake build process.   Though clearly I missed the two calls to it outside of machine.C.  !fixup posted to address this.